Makefile

```diff
@@ -5,7 +5,7 @@ TARGET := $(shell echo $${PWD\#\#*/})
 .DEFAULT_GOAL: $(TARGET)
 
 # These will be provided to the target
-BUILD := `git rev-parse HEAD`
+BUILD := "`cat VERSION`-`date +%Y%m%d-%H%M%S`+`git rev-parse --short HEAD`"
```

Comment:
not trimming trailing newlines could be dangerous. could you fix this please? :)